packages/components-js/tests/vrt/playwright/scripts/adjustRegressionFilenames.ts

```diff
@@ -41,8 +41,19 @@ const walk = (dir: any, done: any): void => {
   });
 };
 
+const deleteRetryDirectories = (source: string): void => {
+  fs.readdirSync(source, { withFileTypes: true })
+    .filter((dirent) => dirent.isDirectory())
+    .map((dirent) => {
+      if (/retry\d+$/.test(dirent.name)) {
+        fs.rmSync(`./${dirent.path}/${dirent.name}`, { recursive: true, force: true });
+      }
+    });
+};
+
```

Comment:
At least in CI this script does not seem to be executed or work?

Reply:
The CI doesn't execute the post script, also didn't in the past. Do you have an idea why?

Reply:
I've created a bug issue: https://github.com/porsche-design-system/porsche-design-system/issues/2897